Detectors/TRD/pid/include/TRDPID/Dummy.h

```diff
@@ -34,13 +34,13 @@ class Dummy final : public PIDBase
   using PIDBase::PIDBase;
 
  public:
-  ~Dummy() final = default;
+  ~Dummy() = default;
 
   /// Do absolutely nothing.
   void init(o2::framework::ProcessingContext& pc) final{};
 
   /// Everything below 0.f indicates nothing available.
-  PIDValue process(const TrackTRD& trk, const o2::globaltracking::RecoContainer& input, bool isTPC) final
+  [[nodiscard]] float process(const TrackTRD& trk, const o2::globaltracking::RecoContainer& input, bool isTPCTRD) const final
```

Comment:
Why the `[[nodiscard]]` here and in the other places below? I have never seen it used in O2 and I don't know if it is of value here. Could you please elaborate? I don't think it improves the readability of the code

Reply:
I think it makes the code more readable and says that this value should not be ignored, but I will delete it anyways.